README.md

```diff
@@ -23,21 +23,39 @@ Potential benefits:
 
 Django-components can be particularly useful for larger Django projects that require a more structured approach to UI development, without necessitating a shift to a separate frontend framework.
 
-## Summary
+## Quickstart
 
-It lets you create "template components", that contains both the template, the Javascript and the CSS needed to generate the front end code you need for a modern app. Use components like this:
+django-components lets you create reusable blocks of code needed to generate the front end code you need for a modern app. 
+
+Define a component in `components/calendar/calendar.py` like this:
+```python
+@register("calendar")
+class Calendar(Component):
+    template_name = "template.html"
+
+    def get_context_data(self, date):
+        return {"date": date}
+```
+
+With this `template.html` file:
+
+```htmldjango
+<div class="calendar-component">Today's date is <span>{{ date }}</span></div>
+```
+
+Use the component like this:
 
 ```htmldjango
-{% component "calendar" date="2015-06-19" %}{% endcomponent %}
+{% component "calendar" date="2024-11-06" %}{% endcomponent %}
 ```
 
-And this is what gets rendered (plus the CSS and Javascript you've specified):
+And this is what gets rendered:
 
 ```html
-<div class="calendar-component">Today's date is <span>2015-06-19</span></div>
+<div class="calendar-component">Today's date is <span>2024-11-06</span></div>
 ```
 
-[See the example project](https://github.com/EmilStenstrom/django-components/tree/master/sampleproject) or read on to learn about the details!
```

Comment:
I'd keep the link to the example project, because this is the only prominent place where it's mentioned.

Reply:
Yeah, but where? I don't think it should be part of the quickstart, it's way to hard to differentiate the stuff in there when you hardly know what the projects is about. I think it's more of an advanced resource for config issues...

Reply:
What do you think of this?

```markdown
Read on to learn about the details!

If you instead prefer to jump right into the code, [check out the example project](...)
```

I agree that for someone who comes across django-components for the first time, they wouldn't find the example useful. But I also think that as soon as they start the installation steps, or then when they're learning how to define components, or later to remind themselves of what the correct syntax of something is... In all those cases they might want to check how things are implemented in an actual (demo) project. And since it's something that they might repeatedly come back to, that's why I feel like it should be somewhere high up in an obvious place.